llvm/test/CodeGen/AArch64/neon-dotreduce.ll

```diff
@@ -860,19 +1213,253 @@ entry:
   ret i32 %x
 }
 
-define i32 @test_udot_v24i8(ptr nocapture readonly %a, ptr nocapture readonly %b, i32 %sum) {
-; CHECK-SD-LABEL: test_udot_v24i8:
+
```

Comment:
I didn't change the original tests; that's `git diff` doing a poor job. Running `git diff --minimal` gives better representation